showpass/Showpass-E2E-tests
# Payment 'Card Information' and 'Billing Address' sections appearance and inline error validation

**Description:**
On the ‘Payment Methods’ page (Payments) there are two sections, two forms. One is the ‘Card Information’ section and the other is ‘Billing Address’.
This script will verify the forms appearance including the presence of inline validation errors.

Steps:
- Navigate to https://beta.showpass.com/account/profile/update-payment/
- Go to the ‘Payment’ page
- Make sure two forms showed up ‘Card Information’ and ‘Billing Address’
- Verify that all labels exist and are visible
- Verify that the input fields are present
- Click ‘Save’ button and verify inline validation errors in both sections

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-16

**How to test:**
Run the following specs and ensure they pass: (npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js")
- "checking the 'Card Information' form appearance"
- "checking the 'Billing Address' form appearance"

**Test execution results:**
<img width="868" alt="Tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/46987e3a-ee19-413e-9706-6d04c24d495c">
